src/Plug.vala

```diff
@@ -96,13 +96,14 @@ public class ApplicationsPlug : Switchboard.Plug {
         search_results.set ("%s → %s".printf (display_name, _("Startup")), STARTUP);
         search_results.set ("%s → %s".printf (display_name, _("Default Apps")), DEFAULTS);
         search_results.set ("%s → %s".printf (display_name, _("Default Application")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Web Browser")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Music Player")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Email Client")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Text Editor")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default File Browser")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Video Player")), DEFAULTS);
-        search_results.set ("%s → %s".printf (display_name, _("Default Calendar")), DEFAULTS);
+        search_results.set ("%s → %s → %s".printf (display_name, _("Default"), _("Web Browser")), DEFAULTS);
```

Comment:
While we're changing these, can we put them in alphabetical order?

Reply:
I could, but my logic was following the order they appear in the UI, that makes it a bit easier to compare them. But I can totally change it to be alphabetically.

Reply:
Ah yeah. I think we probably need to redesign this page in the UI. It seems like a pretty random ordering. I feel like alphabetical would make it easier to find a specific line regardless of what the UI side looks like, but I can see wanting to try to mirror the UI as well. In that case, I think I'm fine with whichever